examples/format_example_modpack.modip.json

```diff
@@ -0,0 +1,175 @@
+      "caption": "This is an example embed."
+    },
+    {
+      "rel": "gallery",
+      "type": "video",
+      "src": "https://example.com/video.webm",
+      "caption": "This is an example direct video link.",
+      "sha256": "< sum goes here >"
+    },
+    {
+      "rel": "gallery",
+      "type": "image",
+      "src": "https://example.com/image.png",
+      "caption": "This is an example image."
+    }
+  ],
+  "authors": [
+    {
+      "team": true,
+      "id": "111111-111111-111111-111111-111111",
+      "name": "MODIP Contributors",
+      "uri": "https://github.com/modip-org",
+      "members": [
+        {
+          "id": "111111-111111-111111-111111-111111",
+          "name": "Example MODIP Member",
+          "uri": "https://example.com/example-webpage"
+        }
+      ]
+    }
+  ],
+  "tags": [
+    "modpack"
+  ],
+  "license": {
+    "id": "MIT",
+    "name": "MIT",
+    "uri": "https://opensource.org/licenses/MIT",
+    "modpackUsage": "alwaysAllow"
+  },
+  "links": [
+    {
+      "rel": [
+        "x-discord",
+        "community"
+      ],
+      "name": "Modpack Discord",
+      "uri": "https://discord.gg/example"
+    }
+  ],
+  "versions": [
+    {
+      "id": "1.0.0",
+      "name": "Release 1.0.0",
+      "semver": "1.0.0",
+      "releaseDate": "2020-04-08T12:00:00Z",
+      "changelog": "Example changelog goes here... <b>HTML</b> may be used.",
+      "dependencies": [
+        {
+          "id": "minecraft",
+          "version": [
+            "1.16.1",
+            "1.16"
+          ]
+        },
+        {
+          "id": "framework-fabric-loader",
+          "required": true,
+          "version": ">= 0.6.1.45"
+        },
+        {
+          "id": "example-mod",
+          "required": true,
+          "version": [
+            "1.0.0",
+            "1.0.1"
+          ]
+        },
+        {
+          "id": "client-only-mod",
+          "required": true,
+          "_comment": [
+            "Below is an example *Condition*",
+            "This dependency is only allowed on the client, not the server"
+          ],
+          "allowed": {
+            "environment": {
+              "client": true,
+              "server": false
+            }
+          },
+          "version": "3.0.0"
+        },
+        {
+          "id": "client-mod-with-optional-server",
+          "_comment": [
+            "This mod is required on the client,",
+            "but optional on the server"
+          ],
+          "required": {
+            "environment": {
+              "client": true,
+              "server": false
+            }
+          },
+          "version": "3.0.0"
+        },
+        {
+          "src": "https://example.com/external-mod.json",
+          "id": "external-mod",
+          "required": false,
+          "version": "6.2.1"
+        }
+      ],
+      "installation": {
+        "method": "instanceInstall"
+      },
+      "files": [
+        {
+          "rel": "compressedOverrides",
```

Comment:
This would be a good case for an installation method extractZip. But then what should rel be? And what is the installation method for the overall pack? Hmm. Perhaps installation methods should apply to files instead?

Reply:
Hmm. An `extractZip` installation method could be useful. But there does need to be a way to indicate modpacks vs mods/resourcepacks/etc, because they are handled very differently. I have thought about per-file installation methods before, and that could work with `extractZip` applied to the overrides.